tests/config/state/tests.yml

```diff
@@ -9,3 +9,6 @@ components:
     allOperations: true
   - component: sqlserver
     allOperations: true
+  - component: postgresql
+    allOperations: false
+    operations: [ "set", "get", "delete", "bulkset", "bulkdelete", "transaction", "etag" ]
```

Comment:
Nit: add newline at end of file for valid Linux text file

Reply:
fixed.